src/Home.js

```diff
@@ -46,14 +57,24 @@ function Home () {
     console.log(`Total: ${allGirls.length}, removed: ${removedGirls.current.length}.`)
 
     if ( shouldFetch() && ( !isFetching || !isFetchingNextPage ) ) {
-      console.log(`GET MORE GIRLS! Total: ${allGirls.length}, removed: ${removedGirls.current.length}.`)
-      fetchNextPage()
+      console.log(`GET MORE GIRLS! Total: ${allGirls.length}, removed: ${removedGirls.current.length}.`);
+      sendEvent({
+        category: 'interaction',
+        action: 'refetch'
```

Comment:
Actually it'd be interesting if we can also send a label like `page_3` or `page_4`, so later in GA we can group / rank the refetch by the page number. I think this is optional

Reply:
that's too much for now, but we could count refetch, how about that?